sophiefitzpatrick/world-kind
# details in commit description

This PR closes issue #1 and #4 

In this PR, I have
- set up the inital gatsby site
- added styled components
- added a layout component and bare bones page
- added a global css file

To test: 
- pull the changes
- run `gatsby develop` in the root of the `worldkind` folder (not `world-kind`) - you should be able to view this site at `localhost:8000`

<img width="1440" alt="Screenshot 2019-08-10 16 42 41" src="https://user-images.githubusercontent.com/26777813/62823853-ee0b1f80-bb8d-11e9-9168-23a70c661cfc.png">
